json-schemas/src/client-events-api-requests.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably API Requests Client Event",
+  "type": "object",
+  "properties": {
+    "host": {
+      "type": "string",
+      "description": "the host the request was made to"
+    },
+    "requestId": {
+      "type": "string",
+      "description": "the unique ID of the request that Ably can use to correlate a request event with internal logs, if necessary"
+    },
+    "region": {
+      "type": "string",
+      "description": "the region the request was made in"
```

Comment:
```
The region the request was processed in.
```

Reply:
And I just looked at a sample message, and it seems this is in fact a site, not a region (i.e. it has a `-A` suffix):

```
"region": "eu-central-1-A"
```

Should we change it to `site` now before we publish this schema?

Reply:
That would make sense, I'll also change the code in realtime to do that.